azurelinuxagent/ga/exthandlers.py

```diff
@@ -195,9 +195,7 @@ def handle_ext_handlers(self, etag=None):
             logger.verbose("No ext handler config found")
             return
 
-        in_vm_artifacts_profile = self.protocol.get_in_vm_artifacts_profile()
-        if in_vm_artifacts_profile and \
-           in_vm_artifacts_profile.is_extension_handlers_handling_on_hold():
+        if self.protocol.is_ext_handling_on_hold():
```

Comment:
Does this method need to be implemented or stubbed out in the Azure Stack / metadata protocol as well?